src/dc_configure.c

```diff
@@ -722,34 +722,49 @@ void dc_job_do_DC_JOB_CONFIGURE_IMAP(dc_context_t* context, dc_job_t* job)
 
 
 /**
- * Configure and connect a context.
- * For this, the function creates a job that is executed in the IMAP-thread then;
+ * Configure a context.
+ * For this purpose, the function creates a job
+ * that is executed in the IMAP-thread then;
  * this requires to call dc_perform_imap_jobs() regularly.
  *
- * - Before you call this function, you should set at least `addr` and `mail_pw`
- *   using dc_set_config().
+ * - Before you call this function,
+ *   you must set at least `addr` and `mail_pw` using dc_set_config().
  *
- * - Use `mail_user` to use a different user name than `addr` and `send_pw` to
- *   use a different password for the SMTP server (otherwise `mail_pw` is used)
+ * - Use `mail_user` to use a different user name than `addr`
+ *   and `send_pw` to use a different password for the SMTP server.
  *
- *     - If _no_ more options are specified, the function tries to get these
- *       **using autoconfigure/autodiscover**.
+ *     - If _no_ more options are specified,
+ *       the function **uses autoconfigure/autodiscover**
+ *       to get the full configuration from well-known URLs.
  *
- *     - If _more_ options as `mail_server`, `mail_port`, `send_server`, `send_port`, `send_user` or `server_flags`
- *       are specified, **autoconfigure/autodiscover is skipped**.
+ *     - If _more_ options as `mail_server`, `mail_port`, `send_server`,
+ *       `send_port`, `send_user` or `server_flags` are specified,
+ *       **autoconfigure/autodiscover is skipped**.
  *
- * - While dc_configure() returns immediately, the started configuration-job may take a while,
- *   you can stop it using dc_stop_ongoing_process().
+ * While dc_configure() returns immediately,
+ * the started configuration-job may take a while.
+ * You can stop it using dc_stop_ongoing_process().
```

Comment:
What does a second call to dc_configure() do? Does it simply not execute or does it put a second job to imap?  
why does dc_configure() not call dc_stop_ongoing_process() impilicitely?

Reply:
and if the context/db is already configured then dc_configure is a no-op, right? this should be mentioned.

Reply:
> What does a second call to dc_configure() do? Does it simply not execute or does it put a second job to imap?

it puts a second job to imap. 
does not make sense, however.

> why does dc_configure() not call dc_stop_ongoing_process() impilicitely?

it does. an explicit call is only needed to stop an ongoing configuration job before it is finished.

> and if the context/db is already configured then dc_configure is a no-op, right? this should be mentioned.

no - it just configures again. this is useful eg. if the user wants to change some settings, password, options, whatever.

Reply:
refined the doc and avoid enquing a second config-job while the running is not yet finished.